CODING_GUIDELINES.md

```diff
@@ -43,12 +48,12 @@ Use undefined. Do not use null.
 
 ## Comments
 
-Use JSDoc style comments for functions, interfaces, enums, and classes. [Create JSDoc Comments for JavaScript IntelliSense](https://docs.microsoft.com/en-us/visualstudio/ide/create-jsdoc-comments-for-javascript-intellisense).
+Use JSDoc style comments for functions, interfaces, enums, and classes. [Create JSDoc Comments for JavaScript IntelliSense](https://docs.microsoft.com/visualstudio/ide/create-jsdoc-comments-for-javascript-intellisense).
 
 ## Strings
 
-* Use double quotes for strings.
-* All strings visible to the user need to be localized (make an entry in diagnosticMessages.json).
+* Use single quotes for strings.
```

Comment:
It's not that I care; but why singles?

Reply:
https://basarat.gitbook.io/typescript/styleguide#quotes

Reply:
oh well :P